README.md

```diff
@@ -1,2 +1,54 @@
-# soen342
-A repository for the soen342 project.
+# SOEN342
+A repository for the Craigstutors project.
+
+## Objectives
+Our goal is to create a site to aid students in the best way possible through expert tutelage. Craigstutors is an e-learning platform that allows elementary students to book tutors on a variety of subjects. Our AI-powered personality-matching algorithm rates your child’s compatibility with that of a tutor in order to provide our students with the best possible assistance.
+
+## Core Features
+There are four types of users: Guests, Students, Tutors, and Administrators
+
+Guests:
+* Guest users can sign up and create an account, as either a student, tutor or administrator.
+* Access to the homepage to view featured courses, ongoing promotions, featured reviews, and featured tutors.
+* Access to the list of all courses and subjects, but cannot sign up to any of them prior to registration.
+* Access and search through the list of tutors.
+* Access to the “About Us” page to find more information on the services’ goals, staff, background, etc.
+* Access a contact page where they can find our contact information, such as our email address and phone number, as well as a live chat section where an automated Q&A bot or support staff can answer questions live.
+* Guest users with managerial experience may apply to join the admin team and gain access to the “Careers” section.
+* Guests can access the FAQs page to see frequently asked questions and their answers.
+
+Students:
+* Students can register for lessons/tutors using an automatic matching system.
+* Students can fill out a timesheet where they can select their weekly availability, filter the subjects they wish to be tutored in, the language they wish to be taught in and their school grade.
+* Students will be prompted to take a quick personality test to determine the tutors that best match their personality.
+* Students can see the course plan for each session and can archive their lesson material and documents.
+* Students have access to a built-in video call platform,where the students and tutors may have their tutoring sessions.
```

Comment:
LGTM.
Just nitpicking here for the sake of grammatical correctness; can we add a space after the comma? Otherwise, everything is good.